library/SimplePie.php

```diff
@@ -1218,6 +1225,15 @@ public function strip_htmltags($tags = '', $encode = null)
 		}
 	}
 
+	public function rename_attributes($tags = '', $encode = null)
+	{
+		if ($tags === '')
```

Comment:
Why not `null`? The param could then have `string[]|null` type.

Reply:
I improved the signature. Second parameter is obsolete.